app/views/vm_common/console_vmrc.html.haml

```diff
@@ -367,36 +367,42 @@
         // Start up VMRC
         vmrc = document.getElementById("vmrc");
 
-        vmNameTooltip = $("#vmNameTooltip");
-        var vmTitle = $("#vmTitle");
+        if (typeof vmrc.isReadyToStart === 'undefined') {
+          window.location.href = "vmrc://clone:#{ticket}@#{host}:443/?moid=#{vmid}";
+          setTimeout(function () { window.close(); }, 5000);
```

Comment:
The old VMRC plugin needs the new window, but the new version opens an external application and this window is no longer needed.